app/Helpers/Helper.php

```diff
@@ -86,9 +86,6 @@ function stationLink($name, $classes = "text-trwl clearfix"): String {
     return $return;
 }
 
-function formatNewDay($DateObject) {
-
-
-
-    return __("dates." . $DateObject->format('l')) .', '. $DateObject->format('j').'. '.__("dates." . $DateObject->format('F')) .' '. $DateObject->format('Y');
+function formatNewDay($dateObject) {
+    return __("dates." . $dateObject->format('l')) .', '. $dateObject->format('j').'. '.__("dates." . $dateObject->format('F')) .' '. $dateObject->format('Y');
```

Comment:
Can you please use whitespaces as seperator in a consistent way?
It's extremely It is very hard to read.

Reply:
You're right. I'll adjust it in a moment.